### ISSUE TITLE
Demo Application Crashes

### ISSUE THREAD

**Issue by ramvdixit** [SimonVT/android-menudrawer#205]
Imported the menudrawer and actionbarsherlock libraries to the application. Fixed a few import errors and it compiled smoothly. Ran it on my Android 10.1 inch tablet with ICS (Android 4.0) running. Every option other than the left drawer and right drawer crashed the app with a message "Unfortunately, MenuDrawerSamples has stopped". Any idea why? Am I doing anything wrong? The Zip download did not contain the actionbarsherlock lib and I downloaded it separately from the sherlock website.

Am I targeting the correct version? I have set the min and max device targets for the actionbarsherlock, menudrawer and the samples to 14 and 19.
- Mahesh Iyer

**Comment by ramvdixit** [comment]
-- Edit --

Hello Simon,

The left, right, top and bottom drawers work perfectly. In fact super smooth and very responsive. However the other samples crash the app. I tried doing it on a 10.1 inch tablet with 4.0.3 ICS, a 5.5 inch 4.3 Jelly Bean and 2 AVDs configured for Nexus 7 Phone and Nexus 10 Tab. The results are the same. Any pointers? I am sure I am doing something wrong and silly. I looked up the closed issues and didn't find anyone complaining of crashing...

Thanks and congratulations on such a great product.
- Mahesh Iyer
